src/content/smart-bulletins/037.md

```diff
@@ -0,0 +1,112 @@
+
+This prohibition is narrow (e.g., affecting social media or advertising services; purchases
+of services that require use of IT on which TikTok may be present) and it is not anticipated
+to affect many purchase cardholder micro-purchase transactions.
+
+It is important to note that the definition of IT used for this ban encompasses not only IT
+owned or operated by agencies, but also “used by a contractor under a contract with the
+executive agency that requires the use” of that IT, whether expressly or “to a significant
+extent in the performance of a service or the furnishing of a product” (M-23-13 at 1-2).
+That definition does not, however, “include any equipment acquired by a federal contractor
+incidental to a federal contract.”
+
+## Action
+
+GSA SmartPay purchase cards shall not be used for purchases that require the use of
+TikTok or require vendor use of IT that contains TikTok, unless an exception is granted in
+accordance with Office of Management and Budget Memorandum M-23-13. Based on a
+Center for Charge Card Management review, agency purchase cardholders rarely acquire
+media services or make purchases through TikTok. As a result, very few purchase card
+transactions (primarily social media advertising and advertising services) are expected to
+be impacted by this prohibition.
+
+However, agencies which routinely purchase social media and advertising services within
+the micro-purchase threshold using a purchase card may want to consider issuing policy
+placing limits on buying these services on the open market to help manage this risk.
+As a best practice, agencies may also consider issuing policy emphasizing the security
+benefits of using existing government online shopping sites as the first source of supply for
+high-risk products and services, such as: GSA Advantage!, Fed Mall, GSA Global Supply,
+category-managed solutions, agency-specific vehicles, etc. The expectation is that the
+cognizant contracting office will ensure the required TikTok prohibition is incorporated into
+the contracts underlying these offerings.
+
+Note that this ban is different from the Section 889 ”covered technology” prohibition, as
+follows:
+- No contractor representations are required;
+- The prohibition is narrowly focused on the use of a single “app” (TikTok) in the
+performance of government work; and
+- The prohibition applies to devices used in the performance of government work,
+
+regardless of ownership. But it does not apply to personally-owned cell phones not
+used in the performance of a government contract or work, or to technology incidental to
+the performance of such work.
+
+Note that the statutory basis for this ban is separate and distinct from the Section 889
+covered technology prohibition. Searching for a TikTok representation in the System for
+Award Management (SAM) or related SAM search tools is not relevant. Similarly, neither
+TikTok nor ByteDance Limited currently appear on the Excluded Parties list in SAM.
```

Comment:
TikTok nor ByteDance Limited currently appears on the Excluded Parties list in SAM.